source/sdk/lang/Backtrace.ooc

```diff
@@ -52,13 +53,19 @@ BacktraceHandler: class {
         } else {
             // fall back on execinfo? still informative
             version (linux || apple) {
-                stderr write("[lang/Backtrace] Falling back on execinfo.. (build extension if you want fancy backtraces)\n")
+                if (!WARNED_ABOUT_FALLBACK) {
```

Comment:
You know, I have no idea. I figured it's because of some dependency in another library, but it should be easy to test (just make the program crash and see if the backtrace still works).